tests/test_marc.py

```diff
@@ -53,11 +53,55 @@ def test_marc_record_all_fields_transform_correctly():
             "Dewey Call Number 3",
         ],
         citation=(
-            "Célébration : 10 siècles de musique de noël. "
+            "Célébration : 10 siècles de musique de noël. Language material. "
             "https://mit.primo.exlibrisgroup.com/discovery/fulldisplay?vid="
             "01MIT_INST:MIT&docid=alma990027185640106761"
         ),
-        content_type=["Not specified"],
+        content_type=["Language material"],
+        contents=[
+            "Die Seejungfrau : sinfonische Dichtung : (Fantasie nach Hans Christian "
+            "Andersen) (44:29) -- Sinfonietta, op. 23 (22:05).",
+            "Wesendonck-Lieder WWV 91 / Richard Wagner (20:45) -- Rückert-Lieder / "
+            "Gustav Mahler (19:33).",
+        ],
+        contributors=[
+            timdex.Contributor(
+                value="Tran, Phong, composer, performer",
+                kind="Main Entry - Personal Name",
+            ),
+            timdex.Contributor(
+                value="Ceeys (Musical group), composer, performer",
+                kind="Main Entry - Corporate Name",
+            ),
+            timdex.Contributor(
+                value="Theory of Cryptography Conference 2008 : New York, N.Y",
```

Comment:
`rstrip` does have the side effect of removing abbreviation punctuation in some cases

Reply:
I redid the contributor `rstrip`, I don't think it's possible to clean up the `711`s without messing up the `700`s and `710`s so I'm prioritizing the formatting of those 2 given the infrequency of the `711`. I also added `rstrip` to `contents` which I think makes it more readable but let me know your thoughts